README.rst

```diff
@@ -18,6 +18,30 @@ Installation
 
     conda install -c conda-forge nc_time_axis
 
+
+Example Code
+------------
+::
+
+    import nc_time_axis
+	import netcdftime
+	import matplotlib.pyplot as plt
+	import random
```

Comment:
Could you also leave an empty line between the groups and an empty line after the imports